jobsdb/backup.go

```diff
@@ -137,7 +137,11 @@ func (jd *Handle) uploadDumps(ctx context.Context, dumps map[string]string) erro
 		path := filePath
 		operation := func() error {
 			if err := jd.uploadTableDump(ctx, wrkId, path); err != nil {
-				jd.logger.Errorf("[JobsDB] :: Failed to upload workspaceId %v. Error: %s", wrkId, err.Error())
+				jd.logger.Errorw(
+					"failed backup for workspaceId",
```

Comment:
I don't see any need to repeat the workspaceId literal in the error's text
```suggestion
					"failed backup",
```